package.json

```diff
@@ -26,46 +27,54 @@
   },
   "homepage": "https://github.com/kognise/water.css",
   "devDependencies": {
+    "@babel/core": "^7.4.5",
+    "@babel/preset-env": "^7.4.5",
+    "@changesets/cli": "^2.7.2",
+    "@ungap/url-search-params": "^0.1.2",
     "autoprefixer": "^9.5.1",
     "browser-sync": "^2.26.3",
     "chalk": "^2.4.2",
+    "clipboard-polyfill": "^2.8.1",
     "cssnano": "^4.1.10",
+    "eslint": "^7.1.0",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-plugin-html": "^6.0.2",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-markdown": "^2.0.0-alpha.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prefer-arrow": "^1.2.1",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
+    "favicon-mode-switcher": "^1.0.4",
     "gulp": "^4.0.0",
+    "gulp-babel": "^8.0.0",
     "gulp-bytediff": "^1.0.0",
     "gulp-filter": "^5.1.0",
     "gulp-flatten": "^0.4.0",
     "gulp-postcss": "^8.0.0",
+    "gulp-posthtml": "^3.0.4",
     "gulp-rename": "^1.4.0",
     "gulp-sizereport": "^1.2.1",
     "gulp-sourcemaps": "^2.6.5",
+    "gulp-terser": "^1.2.0",
+    "htmlnano": "^0.2.3",
+    "husky": "^4.2.5",
     "postcss-color-mod-function": "^3.0.3",
     "postcss-css-variables": "^0.12.0",
     "postcss-import": "^12.0.1",
-    "postcss-inline-svg": "^3.1.1",
-    "standard": "^14.3.1",
-    "stylelint": "^11.0.0",
-    "stylelint-config-standard": "^19.0.0"
+    "postcss-inline-svg": "^4.1.0",
+    "posthtml-include": "^1.2.0",
+    "stylelint": "^13.5.0",
+    "stylelint-config-standard": "^20.0.0",
+    "vue": "^2.6.10"
```

Comment:
I thought we removed vue from the demo page? Maybe worth reviewing the other dependencies too to see if any are unused

Reply:
I went through the dependencies, it looks like the the only other unused package is `@ungap/url-search-params`, unless I'm missing something.

Is there a reason `clipboard-polyfill` and `favicon-mode-switcher` have to be included in the package.json @jonaskuske? They're only being imported in the documentation and it looks like that only adds extra time to environment setup.

Reply:
Nope!
All dependencies (including Vue and urlsearchparams) were always loaded from a CDN, they're only there because it was necessary back when `script.js` used TypeScript / JSdoc.

Reply:
Fixed by b06097ee38d3213161bcac51fd9c1b898a62d470 :)